### DISCUSSION TITLE
"You're eligible to use co pilot for free" option not showing

### DISCUSSION THREAD

**Discussion by quintincodes** [community/community#125623] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am a verified student yet I don't have access to Github Copilot. the "you're eligible to use copilot for free" doesn't show for me. Can someone assist me?
Over here, it shows that my claimed offers include Github copilot but I don't have access yet. 

![image](https://github.com/community/community/assets/170239327/541f74d6-c1f3-4d2d-8acb-4632d31f885d)
![image](https://github.com/community/community/assets/170239327/9a158602-5f7d-4bdf-9734-f5ba71a8c91c)

**Comment by quintincodes** [comment, marked as answer]
it worked , went to your copilot instead of copilot in settings

**Reply by shgnplaatjies** [reply]
please add more details, this is a problem that I'm also facing.

**Reply by shgnplaatjies** [reply]
**Resolved:**
It works after pressing 'Start free trial', the system distinguishes between student trials and otherwise.
https://github.com/github-copilot/free_signup

**Reply by quintincodes** [reply]
i'm glad it worked. Missed the notification

**Reply by quintincodes** [reply]
For clarification for anyone who needs this in the future, to see the "You're eligible to use Copilot For free" message, you need to go here and click on "Your Copilot" . 

![image](https://github.com/community/community/assets/170239327/27369ebf-bffe-4062-9ebc-39ecd55e5a30)

If the method here doesn't work for you, the above method worked for me. Try that. 

![image](https://github.com/community/community/assets/170239327/4b631da5-d178-4903-a650-84a053499ad2)
